cp-quickstart/docker-cloud-statements.sql

```diff
@@ -0,0 +1,6 @@
+CREATE STREAM pageviews WITH (kafka_topic='pageviews', value_format='AVRO');
+CREATE TABLE users WITH (kafka_topic='users', value_format='PROTOBUF', key = 'userid');
+CREATE STREAM pageviews_female AS SELECT users.userid AS userid, pageid, regionid, gender FROM pageviews LEFT JOIN users ON pageviews.userid = users.userid WHERE gender = 'FEMALE';
+CREATE STREAM pageviews_female_like_89 AS SELECT * FROM pageviews_female WHERE regionid LIKE '%_8' OR regionid LIKE '%_9';
+CREATE TABLE pageviews_regions AS SELECT gender, regionid , COUNT(*) AS numusers FROM pageviews_female WINDOW TUMBLING (size 30 second) GROUP BY gender, regionid HAVING COUNT(*) > 1;
+CREATE STREAM accomplished_female_readers WITH (value_format='JSON_SR') AS SELECT * FROM PAGEVIEWS_FEMALE WHERE CAST(SPLIT(PAGEID,'_')[2] as INT) >= 50;
```

Comment:
Why doesn't this need to be included in the configure-acls command?

Reply:
I'm researching this, but I'm not certain the existing and prior usage of the configure-acls command is correct.   When you create a stream / table without specifying a topic, the topic that gets created is prefixed by the physical ksql cluster.   If you run ccloud kafka topic list after running the demo, there is no `PAGEVIEWS_FEMALE` or `pageviews_female_like_89` instead they are topics prefixed with a pksqlc id.

The above stream works just fine, verified in the ccloud UI, so I'm wondering if ksql is able to create the underlying topics with the proper ACLs on its own.

I'm trying to understand the security here more but can't answer now, not sure if you want to hold up the PR for this answer or not.

Reply:
@ybyzek updated to remove unnecessary topics from configure-acls command